2025/yagsl-2025.2.0.json

```diff
@@ -0,0 +1,64 @@
+{
+  "fileName": "yagsl-2025.2.0.json",
+  "name": "YAGSL",
+  "version": "2025.2.0",
+  "frcYear": "2025",
+  "uuid": "1ccce5a4-acd2-4d18-bca3-4b8047188400",
+  "mavenUrls": [
+    "https://broncbotz3481.github.io/YAGSL-Lib/yagsl/repos"
+  ],
+  "jsonUrl": "https://broncbotz3481.github.io/YAGSL-Lib/yagsl/yagsl.json",
+  "javaDependencies": [
+    {
+      "groupId": "swervelib",
+      "artifactId": "YAGSL-java",
+      "version": "2025.2.0"
+    }
+  ],
+  "requires": [
+    {
+      "uuid": "3f48eb8c-50fe-43a6-9cb7-44c86353c4cb",
+      "errorMessage": "REVLib is required!",
+      "offlineFileName": "REVLib-2025.0.0.json",
+      "onlineUrl": "https://software-metadata.revrobotics.com/REVLib-2025.json"
+    },
+    {
+      "uuid": "151ecca8-670b-4026-8160-cdd2679ef2bd",
+      "errorMessage": "ReduxLib is required!",
+      "offlineFileName": "ReduxLib-2025.0.0.json",
+      "onlineUrl": "https://frcsdk.reduxrobotics.com/ReduxLib_2025.json"
+    },
+    {
+      "uuid": "e995de00-2c64-4df5-8831-c1441420ff19",
+      "errorMessage": "Phoenix6 is required!",
+      "offlineFileName": "Phoenix6-25.1.0.json",
+      "onlineUrl": "https://maven.ctr-electronics.com/release/com/ctre/phoenix6/latest/Phoenix6-frc2025-latest.json"
+    },
+    {
+      "uuid": "ab676553-b602-441f-a38d-f1296eff6537",
+      "errorMessage": "Phoenix5 is required!",
+      "offlineFileName": "Phoenix5-5.35.0.json",
+      "onlineUrl": "https://maven.ctr-electronics.com/release/com/ctre/phoenix/Phoenix5-frc2025-latest.json"
+    },
+    {
+      "uuid": "cb311d09-36e9-4143-a032-55bb2b94443b",
+      "errorMessage": "NavX is required!",
```

Comment:
It would probably help to say Studica is required, since many people haven't caught on to the name change

Reply:
I will fix this and it will be included in the next release.